### DISCUSSION TITLE
Please merge with Valheim Plus

### DISCUSSION THREAD

**Discussion by mirddes** [Mydayyy/Valheim-ServerSideMap#24] [General]
I would greatly appreciate it.
Benefits include wider deployment, large developer/user community and less mods for users to install to join public servers (1 would be ideal).

V+ is always looking for more developers and will welcome you with open arms. V+ is already using github so merging efforts will be super easy.

Thank you for this amazing mod

**Comment by JTF195** [comment]
I like ServerSideMap specifically because it doesn't aim to be everything for everyone.

I prefer each mod to do one specific thing, that way if something breaks, or a dev makes a change I don't like, it's not 'all or nothing'

**Comment by JTF195** [comment]
But hey, you'll be pleased to know, the ValheimPlus devs are already working on a set of comparable map and pin sharing features (currently in Alpha, last I looked)

**Comment by Mydayyy** [comment]
Greetings,

I have to admit that I am on the same  side as JTF here. The entire idea behind this mod was born on my relatively small valheim server with a close knit group of people, since the most annoying part about the game was missing out on exploration and the separate pin placing. Apart from this mod we run a vanilla game and unfortunately I don't see us running vp+.

Also I do have the same opinion that I prefer each mod doing one thing. That way I have greater control if a single mod goes in a direction I dont like.

You are right that joining servers with lots of mods is an issue, and I hope there will be a solution other than "just run this one big mod and you're good". Unfortunately the game is extremely clientside based and the server has almost zero authority and I don't think that will be changing.

I'll be converting this issue to a discussion and leave it open. Happy to hear more thoughts tho.

Cheers
Mydayyy
